docs/rules/exports-last.md

```diff
@@ -0,0 +1,48 @@
+# exports-last
+
+This rule reports all export declaration which come before any non-export statements.
```

Comment:
`This rule enforces that all exports are declared at the bottom of the file. This rule will report any export declarations that comes before any non-export statements.`